content/components/radio-group.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Radio group
+description: A radio group is used to render a short list of mutually exclusive options.
```

Comment:
```suggestion
description: Radio group is used to render a short list of mutually exclusive options.
```

very nitpicky suggestion based on new guidelines (that haven't merged yet) - Cole and I are trying to be more consistent with component descriptions (see [here](https://github.com/primer/design/blob/498f2016ed969204e954bce6c939076d3d72774e/content/guides/contribute/documentation.mdx#documentation-preferences))